packages/studio/src/components/Highlighter.tsx

```diff
@@ -23,11 +23,16 @@ export default function Highlighter() {
     if (!rect) {
       return {};
     }
+    const IFrame = document.querySelector("iframe");
```

Comment:
could we pass this iframe via react context (or something similar) instead? we already have a ref for it in IFramePortal

Reply:
Oooh great idea